### PR TITLE
Switch ListQosServers() with newer ListQosServersForTitle()

### PR DESCRIPTION
**Pull request by nishsinghal20**
1. Switch ListQosServers() with newer ListQosServersForTitle()
2. Retarget Visual Studio build tool chain to newer version

### CODE REVIEW

File: build/Windows/TestClientApp.vcxproj [hunk 1]
```diff
@@ -24,13 +24,13 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```
Comment (pgilmorepf) [inline build/Windows/TestClientApp.vcxproj:27]: We cannot drop support for v141.
All changes have to be backwards compatible.

File: build/Windows/TestClientApp.vcxproj.filters [hunk 2]
```diff
@@ -50,11 +50,11 @@
     <ClInclude Include="$(TestSourceDir)\TestApp\TestApp.h">
       <Filter>Header Files</Filter>
     </ClInclude>
-    <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabAuthenticationContext.h">
```
Comment (pgilmorepf) [inline build/Windows/TestClientApp.vcxproj.filters:53]: Don't change this.

File: build/Windows/XPlatCppWindows.vcxproj [hunk 3]
```diff
@@ -115,7 +115,6 @@
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabAuthenticationContext.h" />
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabApiSettings.h" />
     <ClInclude Include="$(SdkSourceDir)\include\playfab\PlayFabBaseModel.h" />
-
```
Comment (pgilmorepf) [inline build/Windows/XPlatCppWindows.vcxproj:118]: These spaces are an artifact of the auto generation, and it's one of the ones we've just accepted.
It is fixable, but the template would become a lot less readable, so we opted to keep the spaces.

Comment (pgilmorepf) [reply build/Windows/XPlatCppWindows.vcxproj:118]: (really, what will happen is that you just won't change this corresponding template file, and this change will be undone)

Comment (Qazplmqwerty) [reply build/Windows/XPlatCppWindows.vcxproj:118]: []()

**Comment by toddbellMSFT** [comment]
This change was accepted in a modified manner awhile ago: https://github.com/PlayFab/XPlatCppSdk/commit/53214705a21a6c46fbeab4d16498eafa60a5f3e1

**Pull request closed without merging**